### PR TITLE
PLANET-5758 Remove jQuery from header.js

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-5758](https://jira.greenpeace.org/browse/PLANET-5758)

I've removed quite a big chunk of the file, these parts did not seem to be used anymore as I couldn't find any use of the classes in our code.

### Testing

You can test everything either on local or on the [test instance](https://www-dev.greenpeace.org/test-deimos), making sure that everything behaves as expected. You should check both the old and new navigation bar and menu, since both should still be working!

### CODE REVIEW

File: assets/src/js/header.js [hunk 1]
```diff
@@ -1,177 +1,146 @@
-export const setupHeader = function($) {
-  'use strict';
+const updateGaAction = (element, elementName) => {
+  element.dataset.gaAction = `${element.ariaExpanded === 'false' ? 'Open' : 'Close'} ${elementName}`;
```
Comment (lithrel) [inline assets/src/js/header.js:2]: I got confused while testing, but this access to aria attributes is [not supported on firefox](https://caniuse.com/?search=ariaExpanded) at the moment :grimacing:  (seems to be [Aria 1.2 recommendations](https://www.w3.org/TR/wai-aria-1.2/#idl-reflection-attribute-values), still recent)
Without it I don't see other ways than the [get/setAttribute](https://w3c.github.io/aria/#idl_example_usage) function that its supposed to replace

**Review by lithrel — CHANGES_REQUESTED** [review]

File: assets/src/js/header.js [hunk 2]
```diff
@@ -1,177 +1,146 @@
...
+  if (element.classList.contains('nav-search-toggle')) {
+    setSearchToggles(element.ariaExpanded === 'true');
+  }
 
-    // Lock scroll when navigation menu is open
-    if (evt.currentTarget.classList.contains('nav-menu-toggle')) {
-      document.body.classList.toggle('no-scroll-nav-open', evt.currentTarget.getAttribute('aria-expanded') === 'true');
+  // We need to focus the search input when showing it
+  const searchInput = document.querySelector('#search_input');
+  if (element.classList.contains('nav-search-toggle') || element.classList.contains('.navbar-search-toggle')) {
```
[the remark this answers is not in the document]
Comment (mleray) [reply assets/src/js/header.js]: Good catch, fixed!

File: assets/src/js/header.js (hunk 1 above)
Comment (mleray) [reply assets/src/js/header.js]: Oopsie 😬 Updated!

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**